inference-engine/src/transformations/src/transformations/hswish_fusion.cpp

```diff
@@ -0,0 +1,75 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/hswish_fusion.hpp"
+
+#include <memory>
+
+#include <ngraph/opsets/opset4.hpp>
+#include <ngraph/rt_info.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+bool check_constant_value(const std::shared_ptr<ngraph::opset4::Constant>& constant, const float value) {
+    if (!constant) {
+        return false;
+    }
+    if (constant->get_element_type() == ngraph::element::f32 || constant->get_element_type() == ngraph::element::f16) {
+        auto data = constant->cast_vector<float>();
+        if (data.size() != 1 || std::fabs(data[0] - value) < std::numeric_limits<float>::epsilon()) {
+            return false;
+        }
+    } else {
+        return false;
+    }
+    return true;
+}
+
+ngraph::pass::HSwishFusionWithRelu::HSwishFusionWithRelu() {
```

Comment:
Add this transformation to inference-engine/src/transformations/src/transformations/common_optimizations/common_optimizations.cpp